libs/cardano-ledger-conformance/src/Test/Cardano/Ledger/Conformance/ExecSpecRule/Conway/Gov.hs

```diff
@@ -0,0 +1,85 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Test.Cardano.Ledger.Conformance.ExecSpecRule.Conway.Gov () where
+
+import Cardano.Ledger.Conway.Core
+import qualified Data.List.NonEmpty as NE
+import qualified Data.Text as T
+import Cardano.Ledger.Conway.Governance
+import Cardano.Ledger.BaseTypes
+import qualified Lib as Agda
+import Cardano.Ledger.Crypto
+import Cardano.Ledger.Conway (Conway)
+import Cardano.Ledger.TxIn (TxId)
+import Cardano.Ledger.Conway.Rules
+import Lens.Micro
+import Test.Cardano.Ledger.Conformance
+import Test.Cardano.Ledger.Constrained.Conway
+import Test.Cardano.Ledger.Imp.Common
+import Test.Cardano.Ledger.Conway.Arbitrary ()
+import Test.Cardano.Ledger.Conformance.ExecSpecRule.Conway.Base ()
+import Data.Bifunctor (first)
+
+instance 
+  Inject 
+    (TxId StandardCrypto, Proposals Conway, EnactState Conway) 
+    (EnactState Conway) 
+  where
+  inject (_, _, x) = x
```

Comment:
I don't quite see how this is equivalent to the `Inject ConwayGovTransContext` instances from earlier. Or did something change which necessitated this?

Reply:
The `ConwayGovTransContext` was introduced in order to provide the `EnactState` to only the Agda side of things, but the same thing can just be achieved with a 3-tuple, which makes for less code overall.

Reply:
Nice!